website/docs/docs/contributing/testing-a-new-adapter.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Testing a new adapter"
+id: "testing-a-new-adapter"
+---
+
+<VersionBlock lastVersion="1.0">
+
+The previous way to test dbt adapter functionality was via a packaged suite of tests, [`pytest-dbt-adapter`](https://github.com/dbt-labs/dbt-adapter-tests). We've since deprecated that approach in favor of the new testing framework outlined in this document.
+
+</VersionBlock>
```

Comment:
Is this callback valuable? Or should we pretend like it never existed?

When we're ready, we will want to officially "deprecate" the `dbt-adapter-tests` repo by adding a note to its README, and then archive it. Similar to what we did for https://github.com/dbt-labs/dbt-integration-tests. (We're running out of names!)

Reply:
I think we might want the deprecation mentioned for a while. We wouldn't want to confuse somebody by its sudden disappearance.